integration/command.go

```diff
@@ -312,6 +318,11 @@ spec:
     - %s
 EOF
 `, namespace, cmd)
+
+	return &command{
+		name: "Run test-pod",
+		cmd:  cmdStr,
+	}
```

Comment:
Almost all the tests checked for `expectedString: "pod/test-pod created\n"`, is it possible to get it back?

Reply:
I added it back. However I'm not 100% sure it's needed. Having a return value == 0 from the invocation of kubectl should be enough, shouldn't it?

Reply:
Indeed:
https://github.com/kinvolk/inspektor-gadget/blob/67fce08e2f7f7fceb6aebdb0499176e071aee8af/integration/command.go#L209